openmdao/utils/tests/test_options_dictionary.py

```diff
@@ -9,6 +9,30 @@ class TestOptionsDict(unittest.TestCase):
     def setUp(self):
         self.dict = OptionsDictionary()
 
+    def test_reprs(self):
+        self.dict.declare('test', types=int, desc='Test integer value')
+        self.dict.declare('flag', default=False, types=bool)
+        self.dict.declare('long_desc', types=str,
+                          desc='This description is ridiculously long and verbose, so much '
+                               'so that it takes up multiple lines in the options table.')
+
+        self.assertEqual(self.dict.__repr__(), self.dict._dict)
+
+        self.assertEqual(self.dict.__str__(width=80), '\n'.join([
+            "========= ============ ================= ================ ======================",
+            "Option    Default      Acceptable Values Acceptable Types Description           ",
+            "========= ============ ================= ================ ======================",
+            "flag      False        N/A               ['bool']                               ",
```

Comment:
Only comment is more of a question: does the acceptable values field need to be tested? Seems like all three test options don't use it.  Other than that question, this stuff looks good.

Reply:
added a couple more options to the test